pkg/relayer/cmd/cmd.go

```diff
@@ -198,7 +205,23 @@ func setupRelayerDependencies(
 		config.NewSupplyQueryClientContextFn(queryNodeGRPCUrl),            // leaf
 		config.NewSupplyTxClientContextFn(queryNodeGRPCUrl, txNodeRPCUrl), // leaf
 		config.NewSupplyDelegationClientFn(),                              // leaf
-		config.NewSupplySharedQueryClientFn(),                             // leaf
+
+		// Setup the params caches and configure them to clear on new blocks.
+		config.NewSupplyParamsCacheFn[sharedtypes.Params](cache.WithNewBlockCacheClearing),
```

Comment:
Why aren't we doing this for all params?

![Screenshot 2025-02-04 at 3 22 42 PM](https://github.com/user-attachments/assets/29339af3-4c85-417b-ba6c-f10bc545f8e1)

Reply:
Some of the params (`tokenomics` and `gateway`) are not used by the `RelayMiner` and don't need to have a cache implemented.

Reply:
#PUC